3-probability/06-normal-approx/notes.qmd

```diff
@@ -0,0 +1,841 @@
+
+```{r}
+#| echo: false
+#| warning: false
+#| fig-width: 4
+#| fig-height: 2
+#| fig-align: center
+
+data.frame(x = seq(-3.5, 3.5, .01)) %>%
+  mutate(y = dnorm(x)) %>%
+  ggplot(aes(x = x, y = y)) +
+  geom_line() +
+  geom_hline(yintercept = 0, color = "grey40") +
+   geom_vline(xintercept = 0, color = "grey40") +
+  labs(x = "x",
+       y = "y = f(x)") +
+  scale_y_continuous(breaks = NULL) +
+  theme_minimal()
+```
+
+If a random variable $X$ follows a normal distribution, we write
+
+$$X \sim \textrm{N}(\mu, \sigma^2)$$
+
+where $\mu$ is the mean of the distribution and $\sigma$ is its standard deviation. (The particular normal distribution shown above is the *standard normal distribution*, where $\mu = 0$ and $\sigma = 1)$. The curve is completely defined by these two numbers and they are called the *parameters* of the distribution. Every normal curve can be matched up to the standard normal by transforming the $x$ to *standard units* which means shifting the curve so it is centered at $0$ (subtracting $\mu$) and scaling the curve so the standard deviation is $1$ (dividing by $\sigma$).
+
+We can calculate the probability of any event related to $X$ by finding the area under the curve corresponding to that event. That includes the probability that $X$ falls within a particular interval. In the table below, we record the probabilities of three such intervals.
+
+| Interval | Area under the normal curve |
+| ---- | ---- |
+| Between -1 and 1 | `r round(pnorm(1)-pnorm(-1), 2)` |
+| Between -2 and 2 | `r round(pnorm(2)-pnorm(-2), 2)` |
+| Between -3 and 3 | `r round(pnorm(3)-pnorm(-3), 3)` |
+
+So *if* we know a particular distribution is similar in shape to the normal distribution, we're able to calculate the probabilities that the random variable falls within a particular interval.
+
+## The Central Limit Theorem
+
+The normal curve is enormously useful because many data distributions are similar to the normal curve (scaled appropriately), so we can use the areas under the normal curve to approximate the areas of the data distributions to figure out proportions. The reason so many data distributions approximately follow a normal distribution is because of one of the most fundamental results in statistics called the Central Limit Theorem. This astounding result says that __sums (and averages) of independent and identically distributed random variables will follow an approximately normal distribution as the sample size grows large__. If we restate this in terms of our box model, this theorem says that, for large enough $n$, if we draw $n$ times from a box of tickets with replacement, then the probability distribution of the sum and the average of the draws will be approximately normal, *regardless* of the distribution of the tickets in the box. This means that the tickets in the box can be anything, but as long as we draw enough times, the sum will have an approximately bell-shaped distribution. 
```

Comment:
You can probably remove "scaled appropriately" since this doesn't specify the "standard normal".

Reply:
okay.

Reply:
I think I have to put back scaling. We have to divide by sd otherwise as n gets large we don't have a distribution.

Reply:
You mean $S$ is normal but when $n$ hits inf variance breaks? I think few students will know what "scaled appropriately" is referring to and will lead to more confusion that clarity for an asymptotia edge case.

Not a big deal, but I stumbled over that while reading and couldn't quite figure out what it meant.

Reply:
yes, exactly. I won't say "scaled appropriately" but maybe shifted to standard units. I don't want to put incorrect stuff.